src/drivers/fluid_oboe.cpp

```diff
@@ -269,5 +297,33 @@ void delete_fluid_oboe_audio_driver(fluid_audio_driver_t *p)
     delete dev;
 }
 
+
+void
+OboeAudioStreamErrorCallback::onErrorAfterClose(AudioStream *stream, Result result)
+{
+    if (dev->error_recovery_mode == 1) { // Stop
+        FLUID_LOG(FLUID_INFO, "Oboe driver encountered an error (such as earphone unplugged). Stopped.");
+        return;
```

Comment:
In "Stop" mode you return here. The stream remains set but is now closed. Is it safe when the stream will be stopped() and closed() again in `delete_fluid_oboe_audio_driver` later on?

Reply:
You're right, it is not safe to leave it assigned. Not it is reset.